healthcare/api-client/v1/dicom/dicomweb_test.py

```diff
@@ -0,0 +1,277 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import sys
+import uuid
+
+# Add datasets for bootstrapping datasets for testing
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'datasets'))  # noqa
+import datasets
+import dicom_stores
+import dicomweb
+
+cloud_region = 'us-central1'
+base_url = 'https://healthcare.googleapis.com/v1'
+project_id = os.environ['GOOGLE_CLOUD_PROJECT']
+
+dataset_id = 'test_dataset-{}'.format(uuid.uuid4())
+dicom_store_id = 'test_dicom_store_{}'.format(uuid.uuid4())
+
+RESOURCES = os.path.join(os.path.dirname(__file__), 'resources')
+dcm_file_name = 'dicom_00000001_000.dcm'
+dcm_file = os.path.join(RESOURCES, dcm_file_name)
+# The study_uid, series_uid, and instance_uid are not assigned by the
+# server and are part of the metadata of dcm_file
+study_uid = '1.3.6.1.4.1.11129.5.5.111396399361969898205364400549799252857604'
+series_uid = '1.3.6.1.4.1.11129.5.5.195628213694300498946760767481291263511724'
+instance_uid = '{}.{}'.format(
+    '1.3.6.1.4.1.11129.5.5',
+    '153751009835107614666834563294684339746480')
+
+
+@pytest.fixture(scope='module')
+def test_dataset():
+    dataset = datasets.create_dataset(
+        project_id,
+        cloud_region,
+        dataset_id)
+
+    yield dataset
+
+    # Clean up
+    datasets.delete_dataset(
+        project_id,
+        cloud_region,
+        dataset_id)
+
+
+@pytest.fixture(scope='module')
+def test_dicom_store():
```

Comment:
Can you also add the retry and try-except.

You may want to do some research on whether we still need to use `requests` session because I think this will change which exceptions to use for the retry and try-except.

Reply:
Done. We still need to use requests for DICOMweb operations.